vocabularies/Analytics.xml

```diff
@@ -176,6 +179,138 @@
+  aggregate(Amount,Currency),
+  Analytics.MultiLevelExpand(
+    Levels=[{"DimensionProperties":["Country"],"AdditionalProperties":["CountryName"]},
+            {"DimensionProperties":["Region"],"AdditionalProperties":["RegionName"]},
+            {"DimensionProperties":["Segment","Industry"],"AdditionalProperties":[]}],
+    Aggregation=["Amount","Currency"],
+    SiblingOrder=[{"Property":"Amount","Descending":true}],
+    ShowLevels=2,
+    ExpandEntries=[{"Entry":["US"],"Levels":0},
+                   {"Entry":["DE","BW"],"Levels":1}]
+  )/concat(aggregate($count as ResultEntriesCount),
+           skip(20)/top(10)))
+```
+</String>
+        </Annotation>
+        <Parameter Name="InputSet" Type="Collection(Edm.EntityType)" Nullable="false">
+          <Annotation Term="Core.Description" String="Entity set to be processed" />
+        </Parameter>
+        <Parameter Name="Levels" Type="Collection(Analytics.MultiLevelExpandLevel)" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of aggregation levels forming a leveled hierarchy" />
+          <Annotation Term="Core.LongDescription">
+            <String>Each element in the collection defines the properties that constitute one level.
+            A property must not be referenced by more than one level.
+            The first element in the collection defines the property names of the coarsest level,
+            the following elements define the property names of consecutively finer-grained aggregation levels.
+            The function result is the leveled hierarchy with these levels in preorder,
+            entries on the finest-grained level cannot be expanded further.
+            The result of this function does not contain a level representing a root or grand total.
+            All referenced properties must be groupable.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="Aggregation" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="Properties to aggregate for all result entries on all levels" />
+          <Annotation Term="Core.LongDescription">
+            <String>All properties in this collection must be custom aggregates.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="SiblingOrder" Type="Collection(Analytics.MultiLevelExpandSiblingOrder)" Nullable="false">
+          <Annotation Term="Core.Description" String="Sort specification to apply to all direct descendants of a given entry (so-called siblings) in the resulting leveled hierarchy" />
+        </Parameter>
+        <Parameter Name="ShowLevels" Type="Edm.Int64" Nullable="false">
+          <Annotation Term="Core.Description" String="Number N of levels to be shown in the initial expansion" />
+          <Annotation Term="Core.LongDescription">
+            <String>The initial expansion shows the first N levels as defined in `Levels` (0 ≤ N ≤ length of `Levels`).
```

Comment:
Or should that be "1 ≤ N ≤ length of `Levels`"?

Reply:
I would like to suggest that we start counting the levels form 1 if we don't include the root level yet. If we later decide that there's a use case to include the root level we could use `ShowLevels : 0` to specify root only.